framework/src/media/OutputHandler.cpp

```diff
@@ -24,60 +24,22 @@
 #include "OutputHandler.h"
 #include "MediaRecorderImpl.h"
 
-#ifndef CONFIG_HANDLER_STREAM_BUFFER_SIZE
-#define CONFIG_HANDLER_STREAM_BUFFER_SIZE 4096
-#endif
-
-#ifndef CONFIG_HANDLER_STREAM_BUFFER_THRESHOLD
-#define CONFIG_HANDLER_STREAM_BUFFER_THRESHOLD 2048
-#endif
-
 namespace media {
 namespace stream {
 
 OutputHandler::OutputHandler() :
-	mOutputDataSource(nullptr),
-	mWorker(0),
-	mIsFlushing(false),
-	mIsWorkerAlive(false)
+	mIsFlushing(false)
 {
+	mWorkerName = "OutputHandler";
```

Comment:
You can define this in header file like a final type, then you don't have to call mWorkerName("StreamWorker"), and setting name of worker.